students/jaemoon/users/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignupView(View):
+    def post(self,request):
+        try:
+            data           = json.loads(request.body)
+            email_regex    = r'^[a-zA-Z0-9.-_]+\@[a-zA-Z0-9.-]+\.[a-zA-Z]+$'
+            password_regex = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}'
+            if re.match(email_regex,data['email']) and re.match(password_regex,data['password']):
```

Comment:
변수지정과 if 문은 분명 결이 다른 로직입니다. 이런 경우 가독성을 위해 중간에 공백 한 줄을 넣어주세요.
```suggestion
            password_regex = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}'
            
            if re.match(email_regex,data['email']) and re.match(password_regex,data['password']):
```